ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/AgingImages.tsx

```diff
@@ -223,10 +240,10 @@ function AgingImages() {
                                                     <TextInput
                                                         aria-label="Image age in days"
                                                         style={{ minWidth: '100px' }}
-                                                        onChange={(val) => {
+                                                        onChange={async (val) => {
                                                             const value = parseInt(val, 10);
                                                             if (!(value >= maxTimeRange)) {
-                                                                dispatch({
+                                                                await dispatch({
```

Comment:
With the change to have this value backed by localStorage, the `dispatch` function now returns a Promise. The async/await here is purely to avoid the eslint "no dangling promises" error.

Is there another pattern that should be used instead here?

This API returns a Promise in order to make it easier to transition to server-side storage in the future if needed, but we currently have no reason to wait on the result of these promises.